search-service/src/test/kotlin/com/egm/stellio/search/service/EntityOperationServiceTests.kt

```diff
@@ -382,4 +382,31 @@ class EntityOperationServiceTests {
         coVerify { entityPayloadService.deleteEntity(firstEntityURI) }
         coVerify { entityPayloadService.deleteEntity(secondEntityURI) }
     }
+
+    @Test
+    fun `batch merge should ask to merge attributes of entities`() = runTest {
+        coEvery {
```

Comment:
why is the test called `...should ask to merge...` ?

Reply:
Because it should call the merge function from entityPayloadHandler.
I can rename it "batch merge should call mergeEntity" to be more accurate.